apps/sasquatch/src/projectDependency/java/com/microsoft/azure/mobile/sasquatch/activities/MainActivity.java

```diff
@@ -0,0 +1,164 @@
+package com.microsoft.azure.mobile.sasquatch.activities;
```

Comment:
We should not fork the entire class, you could do it like this:
* Have ProjectCrashListener (in project source folder) extends the CrashListener that is common to both flavours.
* Define a tiny class with a static method that returns the crash listener (this class would be in 2 different flavours).
* Thus MainActivity remains in 1 flavour and calls the static method to get the listener.